helloworld.py

```diff
@@ -0,0 +1,141 @@
+    aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
+    endpoint_url=endpoint_url,
+    use_ssl=False,
+    verify=False,
+)
+# Store bucket name
+bucket_name = os.getenv("BUCKET_NAME")
+
+object_type = os.getenv("OBJECT_TYPE")
+
+
+i = 0
+
+app = Flask(__name__)
+
+@app.route('/', methods=['POST'])
+def pythonvectordbappceph():
+
+    collection_name = re.sub('-', '_', bucket_name)
+    app.logger.debug(request.data)
+    event_data = json.loads(request.data)
+    object_key = event_data['Records'][0]['s3']['object']['key']
+    app.logger.debug(object_key)
+    if not client.has_collection(collection_name=collection_name):
+        client.create_collection(
+            collection_name=collection_name,
+            dimension=os.getenv("VECTOR_DIMENSION"),
+            enable_dynamic_field=True,
+            metric_type="COSINE",
+            )
+
+    # define different functions below code snippet
+    object_data = s3.get_object(Bucket=bucket_name, Key=object_key)
+    match object_type:
+        case "TEXT":
+            object_content = object_data["Body"].read().decode("utf-8")
+            objectlist = []
+            objectlist.append(object_content)
+            embedding_fn = milvus_model.DefaultEmbeddingFunction() #dimension 768
+            # embedding_fn = milvus_model.dense.SentenceTransformerEmbeddingFunction(model_name='all-MiniLM-L6-v2',device='cpu') # dimension 768
+            vectors = embedding_fn.encode_documents(objectlist)
+
+        case "IMAGE1":
+            object_data = s3.get_object(Bucket=bucket_name, Key=object_key)
+            object_stream = object_data['Body']
+            # dimesnsion 512
+            extractor = FeatureExtractor("resnet34")
+            # issue : RPC error: [insert_rows], <DataNotMatchException: (code=1, message=The Input data type is inconsistent with defined schema, {vector} field should be a float_vector, but got a {<class 'numpy.float32'>} instead.)>
+            vectors = extractor(object_stream)
+
+        case "IMAGE2":
+            extractor = AutoFeatureExtractor.from_pretrained("microsoft/resnet-50")
+            model = AutoModelForImageClassification.from_pretrained("microsoft/resnet-50")
+            inputs = extractor(images=object_content, return_tensors="pt")
+            # dimension 2048
+            outputs = model(**inputs)
+            # issue : RPC error: [insert_rows], <DataNotMatchException: (code=1, message=The Input data type is inconsistent with defined schema,{vector} field should be a float_vector, but got a {<class 'float'> python
+            vectors = outputs[0][0].squeeze().tolist()
+
+        case _:
+            app.logger.error("Unknown object format")
+
+    client.load_collection(collection_name=collection_name)
+
+    app.logger.debug(vectors)
+    global i
+    i = i + 1
+    object_url = endpoint_url+ "/" + bucket_name + "/"+ object_key
+    data = [ {"id": i, "vector": vectors[0], "url": object_url}
```

Comment:
in the example here: https://milvus.io/docs/image_similarity_search.md
they pass the output of `extractor()` as-is. so, in our case it should be `vectors` and not `vectors[0]`